tool/jt.rb

```diff
@@ -1701,33 +1701,37 @@ def install(name)
   def install_jvmci
     raise "Installing JVMCI is only available on Linux and macOS currently" unless LINUX || MAC
 
+    jvmci_version = ENV["JVMCI_VERSION"] || Utilities.jvmci_version
     dir = File.expand_path("..", TRUFFLERUBY_DIR)
     java_home = chdir(dir) do
       if LINUX
-        jvmci_version = ENV["JVMCI_VERSION"] || Utilities.jvmci_version
-        jvmci_grep = "#{dir}/openjdk1.8.0*#{jvmci_version}"
-        if Dir[jvmci_grep].empty?
+        dir_grep = "#{dir}/openjdk1.8.0*#{jvmci_version}"
+        dir_greps = Dir[dir_grep]
+        if dir_greps.empty?
```

Comment:
I think managing an extra state like `dir_greps` is more complex, what's the problem with just using `Dir[dir_grep]` multiple times?

Reply:
It's just duplicated code and nothing changes between the two so why do it twice?

Reply:
Something can change, if the archive was downloaded or extracted.
This makes the extra assignment needed, which I find not so nice and easy to forget.
Performance optimizations like these are irrelevant here.

There is 4 `Dir[dir_glob]` no matter what, so actually this adds more code than before due to the extra assignments.

Reply:
Done.